### ISSUE TITLE
Flutter windows build process failed: \Microsoft.CppCommon.targets(241,5): error MSB8066

### ISSUE THREAD

**Issue by caraido** [flutter/flutter#84727]
I have been using flutter windows successfully to run my app. However, recently I downloaded Android Studio and all of a sudden my flutter windows didn't work. It threw out error as following:

```
Launching lib\main.dart on Windows in debug mode...
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for 'C:\Users\Schwar
tzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\CMakeFiles\3a7123e8b5d134f8b6f9d9b7be4b7e62\flutter_windows.dll.rule;C:\Users\SchwartzLab\PycharmProjects\baha
vior_rig\behavior_gui\build\windows\CMakeFiles\350a25350ac520c3b2e8372c37f12801\flutter_assemble.rule' exited with code 1. [C:\Users\SchwartzLab\PycharmProjects\bahavior_ri
g\behavior_gui\build\windows\flutter\flutter_assemble.vcxproj]
Building Windows application...
Exception: Build process failed.
```
I uninstalled Android Studio and reinstalled Visual Studio 2019 and flutter. However the error persisted. 

This is the flutter doctor:
```
[√] Flutter (Channel dev, 2.3.0-16.0.pre, on Microsoft Windows [Version 10.0.18363.1621], locale en-US)
    • Flutter version 2.3.0-16.0.pre at C:\Users\SchwartzLab\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision fa5883b78e (4 weeks ago), 2021-05-21 13:04:03 -0700
    • Engine revision 2f067fc4c5
    • Dart version 2.14.0 (build 2.14.0-136.0.dev)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\SchwartzLab\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: C:\Users\SchwartzLab\anaconda3\envs\bahavior_rig_\Library\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.6+8-b765.1)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.10.2)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.10.31410.357
    • Windows 10 SDK version 10.0.19041.0

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] VS Code (version 1.57.0)
    • VS Code at C:\Users\SchwartzLab\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.20.0

[√] Connected device (4 available)
    • Windows (desktop)       • windows • windows-x64     • Microsoft Windows [Version 10.0.18363.1621]
    • Windows (UWP) (desktop) • winuwp  • windows-uwp-x64 • 
    • Chrome (web)            • chrome  • web-javascript  • Google Chrome 91.0.4472.101
    • Edge (web)              • edge    • web-javascript  • Microsoft Edge 89.0.774.68

! Doctor found issues in 1 category.

```

The same app builds on another laptop.

**Comment by stuartmorgan** [comment]
Please provide the output with `-v`; that error message is just the summary error that something in the `assemble` step failed, not the underlying error itself.

(We should see if there's something in the verbose output we can update the error regex with.)

**Comment by caraido** [comment]
Here is the verbose output:

```
[  +80 ms] executing: [C:\Users\SchwartzLab\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +452 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +4 ms] fa5883b78e566877613ad1ccb48dd92075cb5c23
[   +2 ms] executing: [C:\Users\SchwartzLab\flutter/] git tag --points-at fa5883b78e566877613ad1ccb48dd92075cb5c23
[  +97 ms] Exit code 0 from: git tag --points-at fa5883b78e566877613ad1ccb48dd92075cb5c23
[   +1 ms] 2.3.0-16.0.pre
[  +10 ms] executing: [C:\Users\SchwartzLab\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +57 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/dev
[   +1 ms] executing: [C:\Users\SchwartzLab\flutter/] git ls-remote --get-url origin
[  +55 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[ +170 ms] executing: [C:\Users\SchwartzLab\flutter/] git rev-parse --abbrev-ref HEAD
[  +68 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms] dev
[  +92 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +46 ms] executing: C:\Users\SchwartzLab\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +57 ms] List of devices attached
[   +5 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +7 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +31 ms] "flutter run" took 176ms.
[  +22 ms] Target file "windows" not found.
[   +2 ms] 
           #0      FlutterCommand.validateCommand (package:flutter_tools/src/runner/flutter_command.dart:1381:9)
           #1      RunCommand.validateCommand (package:flutter_tools/src/commands/run.dart:448:19)
           #2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:1209:11)
           <asynchronous suspension>
           #3      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1102:27)
           <asynchronous suspension>
           #4      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #5      CommandRunner.runCommand (package:args/command_runner.dart:196:13)
           <asynchronous suspension>
           #6      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:284:9)
           <asynchronous suspension>
           #7      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #8      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:232:5)
           <asynchronous suspension>
           #9      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:62:9)
           <asynchronous suspension>
           #10     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #11     main (package:flutter_tools/executable.dart:91:3)
           <asynchronous suspension>
           
           
[ +135 ms] ensureAnalyticsSent: 127ms
[   +4 ms] Running shutdown hooks
[   +1 ms] Shutdown hooks complete
[   +1 ms] exiting with code 1
```
Here it says "target file 'windows' is not found". Does it mean the devices information? Here is the result of running `flutter devices`:

```
4 connected devices:

Windows (desktop)       • windows • windows-x64     • Microsoft Windows [Version 10.0.18363.1621]
Windows (UWP) (desktop) • winuwp  • windows-uwp-x64 •
Chrome (web)            • chrome  • web-javascript  • Google Chrome 91.0.4472.101
Edge (web)              • edge    • web-javascript  • Microsoft Edge 89.0.774.68

```

**Comment by caraido** [comment]
this is the result when I run in verbose:

```
[  +84 ms] executing: [C:\Users\SchwartzLab\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +464 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +4 ms] fa5883b78e566877613ad1ccb48dd92075cb5c23
[   +2 ms] executing: [C:\Users\SchwartzLab\flutter/] git tag --points-at fa5883b78e566877613ad1ccb48dd92075cb5c23
[ +101 ms] Exit code 0 from: git tag --points-at fa5883b78e566877613ad1ccb48dd92075cb5c23
[   +1 ms] 2.3.0-16.0.pre
[  +12 ms] executing: [C:\Users\SchwartzLab\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +72 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms] origin/dev
[        ] executing: [C:\Users\SchwartzLab\flutter/] git ls-remote --get-url origin
[  +57 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[ +207 ms] executing: [C:\Users\SchwartzLab\flutter/] git rev-parse --abbrev-ref HEAD
[  +78 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms] dev
[  +77 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +8 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +82 ms] executing: C:\Users\SchwartzLab\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +61 ms] List of devices attached
[   +6 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +9 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +84 ms] Multiple devices found:
[  +29 ms] Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.18363.1621]
[   +3 ms] Chrome (web)      • chrome  • web-javascript • Google Chrome 91.0.4472.101
[   +1 ms] Edge (web)        • edge    • web-javascript • Microsoft Edge 89.0.774.68
[   +1 ms] [1]: Windows (windows)
[        ] [2]: Chrome (chrome)
[        ] [3]: Edge (edge)
[   +2 ms] Please choose one (To quit, press "q/Q")
[        ] : 
[+2430 ms] 1
[  +32 ms] Skipping pub get: version match.
[  +67 ms] Found plugin video_player at C:\Users\SchwartzLab\flutter\.pub-cache\hosted\pub.dartlang.org\video_player-1.0.1\
[   +5 ms] Found plugin video_player_web at C:\Users\SchwartzLab\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_web-0.1.4+1\
[   +1 ms] Found plugin video_player_web_hls at C:\Users\SchwartzLab\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_web_hls-0.1.8\
[   +4 ms] Found plugin windows_texture_test at C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\windows_texture_test\
[ +116 ms] Found plugin video_player at C:\Users\SchwartzLab\flutter\.pub-cache\hosted\pub.dartlang.org\video_player-1.0.1\
[   +2 ms] Found plugin video_player_web at C:\Users\SchwartzLab\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_web-0.1.4+1\
[   +1 ms] Found plugin video_player_web_hls at C:\Users\SchwartzLab\flutter\.pub-cache\hosted\pub.dartlang.org\video_player_web_hls-0.1.8\
[   +5 ms] Found plugin windows_texture_test at C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\windows_texture_test\
[   +4 ms] Generating C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +121 ms] Initializing file store
[  +10 ms] Skipping target: gen_localizations
[   +4 ms] Skipping target: gen_dart_plugin_registrant
[   +2 ms] Skipping target: _composite
[   +5 ms] complete
[   +4 ms] Launching lib\main.dart on Windows in debug mode...
[   +4 ms] C:\Users\SchwartzLab\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev
C:\Users\SchwartzLab\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
C:\Users\SchwartzLab\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names
--experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill C:\Users\SCHWAR~1\AppData\Local\Temp\flutter_tools.dec8dfd1\flutter_tool.de9c96df\app.dill
--packages C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts
--track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill build\3c113a45063dc6628e68a4111abcacad.cache.dill.track.dill
--enable-experiment=alternative-invalidation-strategy
[  +22 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires
Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[  +38 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires
Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[   +2 ms] [
             {
               "instanceId": "ff0bbffe",
               "installDate": "2021-06-16T17:35:39Z",
               "installationName": "VisualStudio/16.10.2+31410.357",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community",
               "installationVersion": "16.10.31410.357",
               "productId": "Microsoft.VisualStudio.Product.Community",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Community 2019",
               "description": "Powerful IDE, free for students, open-source contributors, and individuals",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-v16.10#16.10.2",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2021-06-16T17:35:39.4291407Z",
               "catalog": {
                 "buildBranch": "d16.10",
                 "buildVersion": "16.10.31410.357",
                 "id": "VisualStudio/16.10.2+31410.357",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.10.2",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "2",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.10.2+31410.357",
                 "requiredEngineVersion": "2.10.2174.31177"
               },
               "properties": {
                 "campaignId": "",
                 "channelManifestId": "VisualStudio.16.Release/16.10.2+31410.357",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +4 ms] Building Windows application...
[  +25 ms] <- compile package:behavior_app/main.dart
[   +4 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S
C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\windows -B build\windows -G Visual Studio 16 2019
[  +87 ms] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.18363.
[  +24 ms] -- Configuring done
[ +473 ms] -- Generating done
[  +29 ms] -- Build files have been written to: C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/build/windows
[  +12 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows
--config Debug --target INSTALL --verbose
[ +127 ms] Microsoft (R) Build Engine version 16.10.2+857e5a733 for .NET Framework
[        ] Copyright (C) Microsoft Corporation. All rights reserved.
[  +42 ms] Build started 6/16/2021 6:17:53 PM.
[ +254 ms] Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[   +2 ms] Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\INSTALL.vcxproj" (1) is building
"C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +17 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +21 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[        ] Done Building Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +113 ms] Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\INSTALL.vcxproj" (1) is building
"C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[   +4 ms] Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\runner\behavior_gui.vcxproj" (4) on node 1 (default targets).
[        ] Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\runner\behavior_gui.vcxproj" (4) is building
"C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[        ] CustomBuild:
[   +1 ms]   "The build of 'C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\CMakeFiles\350a25350ac520c3b2e8372c37f12801\flutter_assemble.rule'
depends on 'C:\USERS\SCHWARTZLAB\PYCHARMPROJECTS\BAHAVIOR_RIG\BEHAVIOR_GUI\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\CMakeFiles\3a7123e8b5d134f8b6f9d9b7be4b7e62\flutter_windows.dll.rule'. The items cannot be
built in parallel."
[  +48 ms]   Generating C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/flutter_windows.dll,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/flutter_export.h,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/flutter_windows.h,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/flutter_messenger.h,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/flutter_plugin_registrar.h,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc,
C:/Users/SchwartzLab/PycharmProjects/bahavior_rig/behavior_gui/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+1376 ms]   [  +60 ms] executing: [C:\Users\SchwartzLab\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +416 ms]   [ +439 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms]   [        ] fa5883b78e566877613ad1ccb48dd92075cb5c23
[        ]   [        ] executing: [C:\Users\SchwartzLab\flutter/] git tag --points-at fa5883b78e566877613ad1ccb48dd92075cb5c23
[  +49 ms]   [  +51 ms] Exit code 0 from: git tag --points-at fa5883b78e566877613ad1ccb48dd92075cb5c23
[        ]   [        ] 2.3.0-16.0.pre
[   +5 ms]   [   +5 ms] executing: [C:\Users\SchwartzLab\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +48 ms]   [  +48 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ]   [        ] origin/dev
[        ]   [        ] executing: [C:\Users\SchwartzLab\flutter/] git ls-remote --get-url origin
[  +32 ms]   [  +33 ms] Exit code 0 from: git ls-remote --get-url origin
[        ]   [        ] https://github.com/flutter/flutter.git
[  +56 ms]   [  +56 ms] executing: [C:\Users\SchwartzLab\flutter/] git rev-parse --abbrev-ref HEAD
[  +57 ms]   [  +57 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ]   [        ] dev
[  +54 ms]   [  +54 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms]   [   +2 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +96 ms]   [ +101 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +1 ms]   [   +2 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +22 ms]   [  +30 ms] Initializing file store
[  +13 ms]   [  +13 ms] Done initializing file store
[  +41 ms]   [  +41 ms] Skipping target: gen_localizations
[  +10 ms]   [  +10 ms] Skipping target: gen_dart_plugin_registrant
[+2141 ms]   [+2141 ms] Skipping target: unpack_windows
[ +452 ms]   [ +452 ms] Skipping target: kernel_snapshot
[   +2 ms]   [   +2 ms] debug_bundle_windows_assets: Starting due to {}
[ +117 ms]   [ +117 ms] Persisting file store
[  +16 ms]   [  +16 ms] Done persisting file store
[   +6 ms]   [   +5 ms] Target debug_bundle_windows_assets failed: FileSystemException: Directory listing failed, path =
'C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\windows_texture_test\example\windows\flutter\ephemeral\.plugin_symlinks\windows_texture_test\example\build\w
indows\plugins\windows_texture_test\windows_texture_test_plugin.dir\Debug\windows_.F3F76867.tlog\*' (OS Error: The system cannot find the path specified.
[   +2 ms]              , errno = 3)
[   +1 ms]              #0      _Directory._fillWithDirectoryListing (dart:io-patch/directory_patch.dart:37:68)
[        ]              #1      _Directory.listSync (dart:io/directory_impl.dart:243:5)
[   +1 ms]              #2      ForwardingDirectory.listSync (package:file/src/forwarding/forwarding_directory.dart:43:12)
[        ]              #3      ForwardingDirectory.listSync (package:file/src/forwarding/forwarding_directory.dart:43:12)
[   +1 ms]              #4      _AssetDirectoryCache.variantsFor (package:flutter_tools/src/asset.dart:949:78)
[        ]              #5      ManifestAssetBundle._parseAssetFromFile (package:flutter_tools/src/asset.dart:785:37)
[        ]              #6      ManifestAssetBundle._parseAssets (package:flutter_tools/src/asset.dart:690:9)
[        ]              #7      ManifestAssetBundle.build (package:flutter_tools/src/asset.dart:246:53)
[        ]              <asynchronous suspension>
[        ]              #8      copyAssets (package:flutter_tools/src/build_system/targets/assets.dart:53:26)
[   +1 ms]              <asynchronous suspension>
[   +1 ms]              #9      BundleWindowsAssets.build (package:flutter_tools/src/build_system/targets/windows.dart:227:29)
[        ]              <asynchronous suspension>
[        ]              #10     _BuildInstance._invokeInternal (package:flutter_tools/src/build_system/build_system.dart:838:9)
[   +1 ms]              <asynchronous suspension>
[        ]              #11     FlutterBuildSystem.build (package:flutter_tools/src/build_system/build_system.dart:605:16)
[        ]              <asynchronous suspension>
[        ]              #12     AssembleCommand.runCommand (package:flutter_tools/src/commands/assemble.dart:314:32)
[   +1 ms]              <asynchronous suspension>
[   +2 ms]              #13     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1102:27)
[        ]              <asynchronous suspension>
[   +1 ms]              #14     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]              <asynchronous suspension>
[        ]              #15     CommandRunner.runCommand (package:args/command_runner.dart:196:13)
[        ]              <asynchronous suspension>
[   +1 ms]              #16     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:284:9)
[        ]              <asynchronous suspension>
[        ]              #17     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]              <asynchronous suspension>
[        ]              #18     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:232:5)
[        ]              <asynchronous suspension>
[        ]              #19     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:62:9)
[        ]              <asynchronous suspension>
[        ]              #20     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]              <asynchronous suspension>
[        ]              #21     main (package:flutter_tools/executable.dart:91:3)
[        ]   [   +8 ms] "flutter assemble" took 2,963ms.
[        ]              <asynchronous suspension>
[        ]   [   +3 ms]
[        ]              #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
[        ]              #1      AssembleCommand.runCommand (package:flutter_tools/src/commands/assemble.dart:331:7)
[        ]              <asynchronous suspension>
[        ]              #2      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1102:27)
[   +1 ms]              <asynchronous suspension>
[        ]              #3      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[   +1 ms]              <asynchronous suspension>
[        ]              #4      CommandRunner.runCommand (package:args/command_runner.dart:196:13)
[        ]              <asynchronous suspension>
[        ]              #5      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:284:9)
[   +1 ms]              <asynchronous suspension>
[        ]              #6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[   +1 ms]              <asynchronous suspension>
[        ]              #7      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:232:5)
[        ]              <asynchronous suspension>
[        ]              #8      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:62:9)
[        ]              <asynchronous suspension>
[   +2 ms]              #9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]              <asynchronous suspension>
[        ]              #10     main (package:flutter_tools/executable.dart:91:3)
[        ]              <asynchronous suspension>
[  +53 ms]   [  +97 ms] ensureAnalyticsSent: 96ms
[   +1 ms]   [        ] Running shutdown hooks
[        ]   [        ] Shutdown hooks complete
[        ]   [        ] exiting with code 1
[  +77 ms] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for
'C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\CMakeFiles\3a7123e8b5d134f8b6f9d9b7be4b7e62\flutter_windows.dll.rule;C:\Users\SchwartzLab\Pych
armProjects\bahavior_rig\behavior_gui\build\windows\CMakeFiles\350a25350ac520c3b2e8372c37f12801\flutter_assemble.rule' exited with code 1.
[C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\flutter\flutter_assemble.vcxproj]
[   +5 ms] Done Building Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\flutter\flutter_assemble.vcxproj" (default targets) --
FAILED.
[   +7 ms] Done Building Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\runner\behavior_gui.vcxproj" (default targets) -- FAILED.
[  +88 ms] Done Building Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\ALL_BUILD.vcxproj" (default targets) -- FAILED.
[   +1 ms] Done Building Project "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\INSTALL.vcxproj" (default targets) -- FAILED.
[   +2 ms] Build FAILED.
[   +8 ms] "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\INSTALL.vcxproj" (default target) (1) ->
[   +1 ms] "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\ALL_BUILD.vcxproj" (default target) (3) ->
[        ] "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\runner\behavior_gui.vcxproj" (default target) (4) ->
[        ] "C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\flutter\flutter_assemble.vcxproj" (default target) (5) ->
[   +1 ms] (CustomBuild target) ->
[   +2 ms]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for
'C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\CMakeFiles\3a7123e8b5d134f8b6f9d9b7be4b7e62\flutter_windows.dll.rule;C:\Users\SchwartzLab\Pych
armProjects\bahavior_rig\behavior_gui\build\windows\CMakeFiles\350a25350ac520c3b2e8372c37f12801\flutter_assemble.rule' exited with code 1.
[C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\build\windows\flutter\flutter_assemble.vcxproj]
[   +3 ms]     0 Warning(s)
[        ]     1 Error(s)
[   +1 ms] Time Elapsed 00:00:05.82
[  +37 ms] Building Windows application... (completed in 6.7s)
[ +517 ms] Exception: Build process failed.
[   +3 ms] "flutter run" took 10,453ms.
[   +9 ms] 
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
           #1      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:663:9)
           <asynchronous suspension>
           #2      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1102:27)
           <asynchronous suspension>
           #3      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #4      CommandRunner.runCommand (package:args/command_runner.dart:196:13)
           <asynchronous suspension>
           #5      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:284:9)
           <asynchronous suspension>
           #6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #7      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:232:5)
           <asynchronous suspension>
           #8      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:62:9)
           <asynchronous suspension>
           #9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #10     main (package:flutter_tools/executable.dart:91:3)
           <asynchronous suspension>
           
           
[  +23 ms] ensureAnalyticsSent: 19ms
[   +2 ms] Running shutdown hooks
[   +1 ms] Shutdown hooks complete
[   +2 ms] exiting with code 1

```

**Comment by stuartmorgan** [comment]
> Target debug_bundle_windows_assets failed: FileSystemException: Directory listing failed, path = 'C:\Users\SchwartzLab\PycharmProjects\bahavior_rig\behavior_gui\windows_texture_test\example\windows\flutter\ephemeral\.plugin_symlinks\windows_texture_test\example\build\w
indows\plugins\windows_texture_test\windows_texture_test_plugin.dir\Debug\windows_.F3F76867.tlog\*' (OS Error: The system cannot find the path specified.

I'm not sure why it would be looking in a plugin symlink directory during asset bundling. Have you manually added something relating to that directory to your assets in pubspec.yaml?

**Comment by TahaTesser** [comment]
Hi @caraido 
Thanks for filing the issue, please provide information asked above

**Comment by darshankawar** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
